### PR TITLE
In shortcuts dialog tailor tooltip to item under cursor

### PR DESCRIPTION
**Pull request by dterrahe**
Provide more specific information for shortcuts that have multiple elements/effects and those that have fallbacks. Also make sure that for all actions that can be assigned a shortcut, the "type" column has been filled in.

image

### CODE REVIEW

File: src/gui/accelerators.c [hunk 1]
```diff
@@ -956,24 +957,38 @@ gboolean dt_shortcut_tooltip_callback(GtkWidget *widget,
...
+                                            _("start typing for incremental search"),
+                                            _("click to filter shortcuts list"),
+                                            _highlighted_action ?
+                                            _("\nright click to show action of selected shortcut")
+                                            : "",
+                                            def || action->type > DT_ACTION_TYPE_SECTION ?
+                                            _("\ndouble-click to define new shortcut")
+                                            : "",
+                                            def ?
+                                            "\n\nmultiple shortcuts can be defined for the same action;"
```
Comment (TurboGit) [inline src/gui/accelerators.c:985]: shouldn't the trailing `;` be `:` ?

Comment (dterrahe) [reply src/gui/accelerators.c:985]: If you prefer. It is not mandatory to set different params; you can have identical shortcuts, one from midi for example. Maybe it should just be a full stop. @elstoc?

Comment (elstoc) [reply src/gui/accelerators.c:985]: I'm not sure it matters. We don't even start new sentences with capital letters.

**Review by TurboGit — APPROVED** [review]
Ok, let's go with that.

**Pull request merged**